### ISSUE TITLE
'Find Next' shortcut F3 and have 'Backward direction' active

### ISSUE THREAD

**Issue by StephanBeckerVW** [notepad-plus-plus/notepad-plus-plus#12154]
### Description of the Issue
If 'Backward direction' option is active in search mask, shortcut F3 is still searching forward while pressing 'Find Next' button is searching correctly in the backward direction.

### Steps to Reproduce the Issue

1. go to the end of the document
2. open search mask by pressing 'STRG + F'
3. enter a search term
4. enable 'Backward direction'
6. press 'F3'

### Expected Behavior
F3 is also searching in the backward direction because it's the shortcut of 'Find Next'

### Actual Behavior
F3 is still searching in forward direction

### Debug Information
Notepad++ v7.9.1   (32-bit)
Build time : Nov  2 2020 - 01:03:56
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19042.1889
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll JSMinNPP.dll mimeTools.dll NppConverter.dll NppExec.dll NppExport.dll XMLTools.dll

**Comment by alankilborn** [comment]
>F3 is also searching in the backward direction because it's the shortcut of 'Find Next'

It's actually NOT and these are two different "Find Next"s, one in the _Find_ window, one in the _Search_ menu.

_Backward direction_ as a checkbox in the _Find_ UI is only supposed to affect the _Find Next_ behavior from the _Find_ window.  F3 behavior is ALWAYS intended to mean "search from caret towards end-of-document".  Correspondingly, Shift+F3 always means "search caret towards start-of-document".

Suggestion:

Turn on 2-button find mode in the _Find_ window, and forget about the _Backward direction_ checkbox entirely (because with that mode on, you'll have find-next and find-previous on 2 separate buttons, each explicitly stating which direction the search will be run):

![image](https://user-images.githubusercontent.com/77065706/189645385-4d726a51-4631-4074-bc7c-c6eba7420b58.png)

**Comment by StephanBeckerVW** [comment]
Thx for the quick answer.

But than it's really confusing. All the configuration of the find window is considered (e.g. Match Case) with the short cut F3 - Find next. But not 'Backward direction'.
Only as a suggestion:
From my point of view, it makes the most sense to remove this option in general and to activate the 2-button mode by default for a clean UI.

**Comment by alankilborn** [comment]
>But than it's really confusing.

Agreed that for the case you bring up, it is.  Confusion might have been prevented by naming the _Find Next_ choice in the _Search_ menu as "Find Toward End of Document", to avoid the duality of "find next".

>remove this option

The _Backward direction_ option?
Well, that is needed for the case in which the user wants to use the _Enter_ key in the _Find_ window to move between searches.  _Enter_ will respect that direction.

>activate the 2-button mode by default

The 2-button capability is rather a "new" feature to Notepad++.  When something new is added, historically the existing behavior is retained as the default and the attitude is "users can enable new functionality if they like".  To me this is sometimes the wrong attitude, as users will often not know new functionality exists.  I'd opt for "give them the new functionality, they can figure out or ask how to go back to the old way".

**Comment by StephanBeckerVW** [comment]
Thx for the explanation.
I now know how do deal with it.
Can be closed.

**Comment by alankilborn** [comment]
>Can be closed.

As the petitioner, YOU are the best person to close it.  There should be a Close button for you.

**Comment by StephanBeckerVW** [comment]
It's now more clear.
